application/src/main/java/org/wecancodeit/serverside/model/Topics.java

```diff
@@ -7,7 +7,7 @@
 import java.util.Collection;
 
 @Entity
-public class Topic {
+public class Topics {
```

Comment:
I don't love this name change, can we look into just getting the import to work?